src/services/hsm.service.js

```diff
@@ -141,78 +343,57 @@ async function derivateHtrCkd(
     }
   }
 
-  // Derivation 1: Bip Code
-  await hsmConnection.blockchain.createBip32ChildKeyDerivation(
-    derivationVersion, // Version
-    hsm.enums.BCHAIN_SECURE_BIP32_INDEX.BASE + 44, // Derivation index
-    false, // Exportable
-    true, // Temporary
-    hsmKeyName, // Parent key name
-    childKeyNames.HTR_CKD_BIP_KEYNAME, // Child key name
-  );
+  if (!acctPathExists) {
+    // derive from root to account path
+    const childKeyNames = {
+      HTR_CKD_BIP_KEYNAME: 'HTR_BIP_KEY',
+      HTR_CKD_COIN_KEYNAME: 'HTR_COIN_KEY',
+    };
```

Comment:
Should this be in the constants file?

Reply:
These are temporary key names that are used only once during the first start of the wallet.
So I think its safe to have them here.